docs/reference/ml/ml-shared.asciidoc

```diff
@@ -284,6 +283,14 @@ include::{docdir}/ml/ml-shared.asciidoc[tag=mode]
 include::{docdir}/ml/ml-shared.asciidoc[tag=time-span]
 end::chunking-config[]
 
+tag::class-assignment-objective[]
+Defines the objective to optimize when assigning class labels. Available
+objectives are maximize_accuracy and maximize_minimum_recall. When maximizing
```

Comment:
Could we wrap the enum values in "`" here?

Reply:
[854fdf7](https://github.com/elastic/elasticsearch/pull/52763/commits/854fdf734bb5841364b098c79fb9a44e1b26e2eb)